src/backend/commands/explain_gp.c

```diff
@@ -2166,7 +2166,7 @@ gpexplain_formatSlicesOutput(struct CdbExplain_ShowStatCtx *showstatctx,
             }
             else
             {
-                ExplainPropertyInteger("Executor Memory", ss->peakmemused.vmax, es);
+                ExplainPropertyFloat("Executor Memory", ss->peakmemused.vmax, 0, es);
```

Comment:
why not as in https://github.com/arenadata/gpdb/blob/d26e12a16a3cb00888c63fedd66046231ecc8fbb/src/backend/commands/explain_gp.c#L1621

Reply:
I thought it would be better to avoid unnecessary type conversion. But in general, we can use `ExplainPropertyLong`. Do you think it's would be better to use `ExplainPropertyLong`? Or do you suggest converting to KB?

Reply:
I would prefer the 6X code to be closer to the 7X.

Reply:
Okay, but I think we definitely shouldn't change the units of measurement as it could break user applications.